flink-connectors/flink-sql-connector-hbase/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,27 @@
+flink-sql-connector-hbase
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- commons-codec:commons-codec:1.10
+- commons-configuration:commons-configuration:1.7
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- com.google.guava:guava:12.0.1
+- com.yammer.metrics:metrics-core:2.2.0
+- io.netty:netty-all:4.1.44.Final
```

Comment:
Shall we list all the netty dependencies if `netty-all` is included? 

```
io.netty:netty-codec-smtp:4.1.44.Final
io.netty:netty-transport-native-epoll:4.1.44.Final
io.netty:netty-codec-redis:4.1.44.Final
io.netty:netty-resolver-dns:4.1.44.Final
io.netty:netty-codec-dns:4.1.44.Final
io.netty:netty-resolver-dns-native-macos:4.1.44.Final
io.netty:netty-codec-mqtt:4.1.44.Final
io.netty:netty-codec-http2:4.1.44.Final
io.netty:netty-transport-native-unix-common:4.1.44.Final
io.netty:netty-transport-sctp:4.1.44.Final
io.netty:netty-transport-native-epoll:4.1.44.Final
io.netty:netty-codec-http:4.1.44.Final
io.netty:netty-transport-udt:4.1.44.Final
io.netty:netty-handler-proxy:4.1.44.Final
io.netty:netty-resolver:4.1.44.Final
io.netty:netty-common:4.1.44.Final
io.netty:netty-codec-xml:4.1.44.Final
io.netty:netty-transport:4.1.44.Final
io.netty:netty-codec:4.1.44.Final
io.netty:netty-codec-socks:4.1.44.Final
io.netty:netty-codec-stomp:4.1.44.Final
io.netty:netty-buffer:4.1.44.Final
io.netty:netty-transport-rxtx:4.1.44.Final
io.netty:netty-codec-haproxy:4.1.44.Final
io.netty:netty-codec-haproxy:4.1.44.Final
io.netty:netty-tcnative:2.0.28.Final
io.netty:netty-transport-native-kqueue:4.1.44.Final
io.netty:netty-handler:4.1.44.Final
io.netty:netty-codec-memcache:4.1.44.Final
```

I'm not sure about this, but as there are no other 3rd party dependencies in `netty-all`, maybe it's fine just include `netty-all`.

Reply:
cc @JingsongLi Could you help confirm this?

Reply:
Maybe we need, see hadoop notice https://cwiki.apache.org/confluence/display/HADOOP/Bundled+dependencies

Reply:
I believe we don't need to "expand" the dependencies included in "netty-all". Everything that is included in netty-all is licensed to the netty project (as long as there's nothing shaded etc.)